modules/core/utils/upgrade_nomad_clients.sh

```diff
@@ -0,0 +1,105 @@
+#!/usr/bin/bash
+set -euo pipefail
+
+read -p 'AWS_PROFILE: ' awsProfile
```

Comment:
Don't need to do this. 

If you do something like `AWS_PROFILE=test script.sh`, `script.sh` will still pick it up.

Reply:
For example, if I have a script

```
#!/bin/bash

nomad server members
```

and I run it with `NOMAD_TOKEN=$(cat ~/.nomad-token) NOMAD_ADDR=https://nomad.xxx.xxx ./test.sh`

It still works